src/pages/category/Category.js

```diff
@@ -45,20 +45,20 @@ const Category = () => {
       </ul>
       <div className="itemList">
         <ul className="categoryItem">
-          {items.map(meat => {
+          {items.map(({ id, name, price }) => {
             return (
-              <li key={meat.id}>
+              <li key={id}>
                 <div className="items">
-                  <Link to="/Category/Detail">
+                  <Link key={id} to={`/Category/Detail/${id}`}>
```

Comment:
```suggestion
                  <Link to={`/Category/Detail/${id}`}>
```

key는 최상위 부모한테만 있어도 됩니다!
여기서는 li에만 있으면 되는데 이미 있네요 👀

Reply:
넵 반영했습니다!